### PR TITLE
chore: format with Biome

### PR DESCRIPTION
**Pull request by ematipico**
## Changes

This PR makes use of Biome to format the code

## Testing

The current CI should pass

## Docs

N/A

### CODE REVIEW

File: packages/cloudflare/src/index.ts [hunk 1]
```diff
@@ -555,8 +555,8 @@ export default function createIntegration(args?: Options): AstroIntegration {
 					const winningStrategy = notFoundIsSSR
 						? excludeStrategy
 						: includeStrategyLength <= excludeStrategyLength
-						? includeStrategy
-						: excludeStrategy;
+						  ? includeStrategy
```
Comment (ematipico) [inline packages/cloudflare/src/index.ts:558]: Biome still doesn't support the new ternary format

File: biome.json [hunk 2]
```diff
@@ -1,7 +1,28 @@
 {
   "$schema": "https://biomejs.dev/schemas/1.4.0/schema.json",
+  "files": {
+    "ignore": [
+      "**/dist",
+      "**/node_modules",
+      "**/fixtures",
```
Comment (ematipico) [inline biome.json:7]: Is there a reason why we ignore the fixtures?

Comment (alexanderniebuhr) [reply biome.json:7]: @ematipico there was a reason, but I can't remember it.
I'm happy to enable it for fixtures :)

File: packages/test-utils/test-utils.js [hunk 3]
```diff
@@ -294,7 +294,12 @@ export async function parseCliDevStart(proc) {
 	const messages = stdout
 		.split('\n')
 		.filter((ln) => !!ln.trim())
-		.map((ln) => ln.replace(/[🚀┃]/g, '').replace(/\s+/g, ' ').trim());
+		.map((ln) =>
+			ln
+				.replace(/[🚀┃]/g, '')
+				.replace(/\s+/g, ' ')
+				.trim()
+		);
```
Comment (alexanderniebuhr) [inline packages/test-utils/test-utils.js:303]: nit: I personally dislike that it is not formatted like:

```ts
.map((ln) =>
	ln.replace(/[🚀┃]/g, '').replace(/\s+/g, ' ').trim()
);
```

**Review by alexanderniebuhr — COMMENTED** [review]
I'm a big proponent of this change. Happy to give a final review pass once CI doesn't fail anymore.

**Review by alexanderniebuhr — APPROVED** [review]

**Pull request merged**